### PR TITLE
[RAPPS-DB] Add Eclipse Lun3r v2024-07-14

### PR DESCRIPTION
**Pull request by scropoolISreal**
hell yeaahh

lun3r_reactos

### CODE REVIEW

File: lun3r.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Category = 5
+URLSite = https://eclipse.cx/projects/lun3r
+URLDownload = https://github.com/Eclipse-Community/UXP/releases/download/v2024-07-14/lun3r-33.3.3a1.win32.installer.exe
+SHA1 = 43cbdb61a7e619452a333f62fd70437fc4898912
+SizeBytes = 37486614
+Screenshot1 = https://private-user-images.githubusercontent.com/173210839/380523231-b32961cc-526b-4e3b-a64b-9a72c189852e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAwNTU1MjgsIm5iZiI6MTczMDA1NTIyOCwicGF0aCI6Ii8xNzMyMTA4MzkvMzgwNTIzMjMxLWIzMjk2MWNjLTUyNmItNGUzYi1hNjRiLTlhNzJjMTg5ODUyZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyN1QxODUzNDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03OTY2YjNhMzhmMDMwYzE4NzMyZDc5ZGJhODE4MDE3NDcxNDU5ZjlkY2ZlY2ZkMzlkN2RlM2EzMDM3M2RkOTU1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.DxR0w9XDq6ffEyM0O7F3QR1dIbXmNS0t593lupDzlN0
```
Comment (binarymaster) [inline lun3r.txt:11]: This is not a permanent link (shows 404)

Comment (fiatcoupe20vturbo) [reply lun3r.txt:11]: i will check this

idek how to do it :Dxd

**Comment by marjuanm** [comment]
looks good

File: lun3r.txt (hunk 1 above)
Comment (julenuri) [reply lun3r.txt]: It's private so we can't watch it. :(

Comment (fiatcoupe20vturbo) [reply lun3r.txt]: > It's private so we can't watch it. :(

i fixed now see the commit fr👍

**Review by julenuri — APPROVED** [review]

**Pull request merged**